utility_scripts/liftover_variants.sh

```diff
@@ -25,7 +25,7 @@ bash liftover_variants.sh -p /path/to/picard_singularity_image.sif -i /path/to/G
 
 Requirements:
 - Apptainer (although Singularity should work too, please change the script and adjust apptainer to singularity)
-- Picard singularity image
+- Picard 3.0.0 singularity image, available here: https://download.molgeniscloud.org/downloads/vip/images/
```

Comment:
* See above point regarding image/picard version (though got also refer to "Picard 3.0.0 or higher singularity image")
* Perhaps also add link to Picard docs regarding required resource files for Picard itself that need to be passed as command line arguments here?

Reply:
Disagree. That would require constant monitoring of the picard image on already expensive maintaining of documentation.

As for the picard resource files, I consider this out-of-scope for this PR and more of a comment for the VIP Picard 3.0.0 def file, which is already accepted.